src/common.cu

```diff
@@ -406,6 +415,9 @@ testResult_t startColl(struct threadArgs* args, ncclDataType_t type, ncclRedOp_t
         #if defined(RCCL_BFLOAT16)
         rccl_bfloat16 bf16;
         #endif
+        #if defined(RCCL_FLOAT8)
+        rocblas_f8 fp8_e4m3; rocblas_bf8 fp8_e5m2;
```

Comment:
we should use rccl data types instead of raw rocblas types

Reply:
updated.